runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java

```diff
@@ -90,14 +94,31 @@ public Dimension render(Graphics2D graphics)
 		graphics.fillRect(barX, barY, barWidth, barHeight);
 		graphics.setColor(BAR_FILL_COLOR);
 		graphics.fillRect(barX, barY, progressFill, barHeight);
+
+		if (anyPrayerActive && (config.prayerFlickLocation().equals(PrayerFlickLocation.PRAYER_BAR) || config.prayerFlickLocation().equals(PrayerFlickLocation.BOTH)))
+		{
+			long timeSinceLastTick = Duration.between(startOfLastTick, Instant.now()).toMillis();
+
+			float tickProgress = timeSinceLastTick / 600f;
+			tickProgress = Math.min(tickProgress, 1); //Cap to 1
```

Comment:
Shouldn't this be `(timeSinceLastTick % 600) / 600f`? If the time since last tick is more than 600ms, it should wrap to the beginning, not be set to the end of the bar.

Reply:
`(timeSinceLastTick % 600) / 600f` seems to work fine. I'll implement the change and create the method now.

Reply:
How would I go about returning the multiple values needed for both PrayerBarOverlay.java and PrayerFlickOverlay.java as they are used in both classes?